apps/include/fmt.h

```diff
@@ -41,4 +41,4 @@
 
 int FMT_istext(int format);
 
-#endif /* HEADER_FMT_H_ */
+#endif /* OPENSSL_FMT_H_ */
```

Comment:
NIT: The comment should remove the _ at the end.

Reply:
Thanks, I'll fix it.

Reply:
this is still pending.

Reply:
Hmmm... The change was in the script at some time, but than it got lost again. Thanks for doublechecking.

21e92a429b fixup! Use 'OPENSSL_' as common include guard prefix
43b4439d3b fixup! Add `update-openssl-include-guards` script [to be discarded]